### DISCUSSION TITLE
connect with any user with rhel8

### DISCUSSION THREAD

**Discussion by eabernabe** [neutrinolabs/xrdp#3168] [Q&A]
Does anyone know how I can connect to a RHEL 8 system without having to set up a vncpassword?

I have xrdp installed on RHEL 8 system with TigerVNC and I'm able to sign in but I have to setup a vncpassword for the user beforehand. My ultimate goal is to have any domain user able to sign in thru xrdp from a thin client

**Comment by matt335672** [comment]
@eabernabe - it depends what you are trying to do.

If you're just trying to have sesman create a VNC session for each user (it sounds like you are), you don't need to set a VNC password. Just use the default `sesman.ini` which comes with RHEL 8. The VNC password for each user is then handled by sesman.

If you're trying to get all domain users to connect to the same VNC session (created beforehand), you'll currently need a VNC password.

**Comment by eabernabe** [comment]
Thanks @matt335672  Seems the issue was I had to set the sssd.conf file parameter "access_provider = simple" for it to work. Looks like I need to further troubleshoot " = ad" to get it working how I had it

**Reply by matt335672** [reply]
I remember grappling with sssd as a sysadmin a few years back. The secret seems to be to turn up the sssd logging a fair bit and it should tell you what's it's doing.

With xrdp v0.9.x there are a few issues as a user can have multiple names in an enterprise environment. xrdp v0.10.x is better in this regard.
